pyproject.toml

```
@@ -113,4 +114,9 @@ message_template = "Bump to {new_version}"
 tag_template = "{new_version}"
 
 [[tool.tbump.file]]
-src = "jupyterhub-pytest-plugin/_version.py"
\ No newline at end of file
+src = "jupyterhub-pytest-plugin/_version.py"
+
+# specify project entrypoints to make each plugin discoverable by pytest
+# ref: https://docs.pytest.org/en/latest/how-to/writing_plugins.html#making-your-plugin-installable-by-others
+[project.entry-points.pytest11]
+jupyterhub-spawners-plugin= "jupyterhub-pytest-plugin.jupyterhub_spawners"
```

```suggestion
jupyterhub-spawners-plugin= "jupyterhub_pytest_plugin.jupyterhub_spawners"
```

And then also rename the directory `jupyterhub-pytest-plugin` in the same way.